swagger-schema-official/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "noEmit": true,
```

Comment:
Sorry this is ununituitive. @andy-ms maybe the current message:

```
Use `/// <reference types="..." />` in source files instead of using "types" in tsconfig.
```

should be amended to 

```
Use `/// <reference types="..." />` directives in source files and ensure that the "types" field in your tsconfig is empty.